Installer/CairoShell.nsi

```diff
@@ -0,0 +1,385 @@
+
+; Use Unicode rather than ANSI
+Unicode True
+
+; The file to write
+OutFile "CairoSetup_${ARCBITS}bit.exe"
+
+; The default installation directory
+InstallDir "$PROGRAMFILES${ARCBITS}\Cairo Shell"
+
+; Registry key to check for directory (so if you install again, it will 
+; overwrite the old one automatically)
+InstallDirRegKey HKLM "Software\CairoShell" "Install_Dir"
+
+; Minimum .NET Framework release (4.7.1)
+!define MIN_FRA_RELEASE "461308"
+
+!define MUI_ABORTWARNING
+!include "MUI.nsh"
+
+;--------------------------------
+; Pages
+
+  !define MUI_ICON inst_icon.ico
+  !define MUI_HEADERIMAGE
+  !define MUI_HEADERIMAGE_RIGHT
+  !define MUI_HEADERIMAGE_BITMAP header_img.bmp
+  !define MUI_UNWELCOMEFINISHPAGE_BITMAP left_img.bmp
+  !define MUI_UNICON inst_icon.ico
+  ;!define MUI_COMPONENTSPAGE_SMALLDESC
+  !define MUI_WELCOMEFINISHPAGE_BITMAP left_img.bmp
+  !define MUI_WELCOMEPAGE_TEXT "$(PAGE_Welcome_Text)"
+  !define MUI_WELCOMEPAGE_TITLE_3LINES
+  !define MUI_FINISHPAGE_TITLE_3LINES
+  !define MUI_FINISHPAGE_RUN
+  !define MUI_FINISHPAGE_RUN_TEXT "$(PAGE_Finish_RunText)"
+  !define MUI_FINISHPAGE_RUN_FUNCTION "LaunchCairo"
+  !insertmacro MUI_PAGE_WELCOME
+  !insertmacro MUI_PAGE_LICENSE "License.txt"
+  !insertmacro MUI_PAGE_COMPONENTS
+  !insertmacro MUI_PAGE_INSTFILES
+  !insertmacro MUI_PAGE_FINISH
+  
+  !define MUI_WELCOMEPAGE_TITLE_3LINES
+  !define MUI_FINISHPAGE_TITLE_3LINES
+  !insertmacro MUI_UNPAGE_WELCOME
+  !define MUI_UNCONFIRMPAGE_TEXT_TOP "$(PAGE_UnDir_TopText)"
+  !insertmacro MUI_UNPAGE_CONFIRM
+  !insertmacro MUI_UNPAGE_INSTFILES
+  !insertmacro MUI_UNPAGE_FINISH
+
+;--------------------------------
+; Components
+
+!insertmacro MUI_LANGUAGE "English"
+!insertmacro MUI_LANGUAGE "French"
+
+; The stuff to install
+Section "$(SECT_cairo)" cairo
+
+  SectionIn RO
+
+  ; Check .NET version
+  Call AbortIfBadFramework
+
+  ; Set output path to the installation directory.
+  SetOutPath $INSTDIR
+
+  Call AbortIfRunning
+  
+  ; Put file there
+  DetailPrint "Installing Cairo files"
+  File "..\Cairo Desktop\Build\${ARCNAME}\Release\CairoDesktop.exe"
+  File "..\Cairo Desktop\Build\${ARCNAME}\Release\*.dll"
+
+  CreateDirectory "$INSTDIR\Themes"
+  File /r "..\Cairo Desktop\Build\${ARCNAME}\Release\Themes"
+
+  ; Set shell context to All Users
+  SetShellVarContext all
+
+  ; Start menu shortcuts
+  CreateShortcut "$SMPROGRAMS\Cairo Desktop.lnk" "$INSTDIR\CairoDesktop.exe"
+  
+  ; Write the installation path into the registry
+  WriteRegStr HKLM SOFTWARE\CairoShell "Install_Dir" "$INSTDIR"
+  
+  ; Write the uninstall keys for Windows
+  WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\CairoShell" "DisplayName" "Cairo Desktop Environment"
+  WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\CairoShell" "DisplayIcon" '"$INSTDIR\RemoveCairo.exe"'
+  WriteRegStr HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\CairoShell" "DisplayVersion" "BUILD_VERSION"
```

Comment:
The CI replaces this value with the one generated by nbgv, but [the workflow](https://github.com/cairoshell/cairoshell/blob/master/.github/workflows/build.yml) wasn't updated to replace the value in this file instead of the arch-specific files. If you could update that as part of this PR to avoid regressions that would be great!